app/models/concerns/metadata.rb

```diff
@@ -39,6 +39,11 @@ module Metadata
       index.as :displayable
     end
 
+    # For Merritt ARKs
+    property :merritt_id, predicate: RDF::Vocab::DC.identifier do |index|
+      index.as :displayable
```

Comment:
I think there may be no need for `index.as`, since the only purpose of this is to match the objects in the new importer.

What happens if we just drop this block?